qcodes/dataset/descriptions/rundescriber.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, Any
+
+from qcodes.dataset.descriptions.dependencies import InterDependencies_
+
+
+class RunDescriber:
+    """
+    The object that holds the description of each run in the database. This
+    object serialises itself to a string and is found under the run_description
+    column in the runs table
+
+    Extension of this object is planned for the future, for now it holds the
+    parameter interdependencies. Extensions should be objects that can
+    serialize themselves added as attributes to the RunDescriber , such that
+    the RunDescriber can iteratively serialize its attributes when serializing
+    itself.
+    """
```

Comment:
Should we describe here what has changed in version 1?

Reply:
I guess this information belongs to somewhere in the `versioning` module because the class in this file represents the CURRENT version (and i beleive that there is something like that already there). Does this note "Extension of this object is planned for the future .... " confuse you? if so, i can reword it somehow.

Reply:
i've added RunDescriber version log to `converters.py` module docstring.

Reply:
@astafan8 No the extension part is fine. Cool with a changelog there